Add a note about ssh root password on SL Micro 6 (#9)
SUSE/release-notes-alp: adoc/miscellaneous-micro.adoc

When you run the toolbox script to pull and start the toolbox container, a previous version of the container image is pulled.
This does not influence toolbox funcionality and you can use the toolbox container as needed.

> === Password access as root via ssh disabled

Reviewer: ```suggestion
=== Password access as root via SSH disabled
```

> SLE Micro releases 5.x, password-based root-login was allowed but with SLE Micro 6.0 the default was changed to only allow key-based authentication.

Reviewer: ```suggestion
Previously, it was possible to SSH as root using password-based authentication.
In {productnameshort} 6.0 only key-based authentication is allowed by default.
```

Systems upgraded to 6.0 from 5.x carry over the old behavior. New installations will enforce the new behavior.

> Installing the package "openssh-server-config-rootlogin" restores the old behavior and allows password-based login for the root user.

Reviewer: ```suggestion
Installing the package `openssh-server-config-rootlogin` restores the old behavior and allows password-based login for the root user.
```
